File: Java/02- Relacionando Classes/17- Fornecedor/Medico.java

package com.mycompany.mavenproject7;

public class Medico extends Funcionario{
    private String crm;

    public Medico(String crm, String cpf, String rg, String matricula, Setor setor, double salario, Sexo sexo, EstadoCivil estadocivil, String dataNascimento, String nome, String telefone, String email, Endereco endereco) {
        super(cpf, rg, matricula, setor, salario, sexo, estadocivil, dataNascimento, nome, telefone, email, endereco);
        this.crm = crm;
    }

    public String getCrm() {
        return crm;
    }

    public void setCrm(String crm) {
        this.crm = crm;
    }

    @Override
    public String toString() {
        return super.toString() +
               "\n=====Dados do Médico=====" +
               "\nCRM: " + crm;
    }
    
}

File: Python/04- POO/03- Atividades Pontuadas/01- Atividade/models/enums/trabalho/medico.py
from models.funcionario import Funcionario

class Medico(Funcionario):
    def __init__(self, nome, telefone, email, endereco, cpf, rg, data_nascimento, sexo, matricula, setor, salario, crm: str) -> None:
        super().__init__(nome, telefone, email, endereco, cpf, rg, data_nascimento, sexo, matricula, setor, salario)
        self.crm = crm

    def salario_final(self):
        return self.salario

    def __str__(self) -> str:
        return (f"{super().__str__()}" 
                f"\nCRM: {self.crm}"

               )